librina/wrap/librina.i

```diff
@@ -402,21 +434,22 @@ SWIG_JAVABODY_TYPEWRAPPER(public, public, public, SWIGTYPE)
 DOWNCAST_IPC_EVENT_CONSUMER(eventWait);
 DOWNCAST_IPC_EVENT_CONSUMER(eventPoll);
 DOWNCAST_IPC_EVENT_CONSUMER(eventTimedWait);
-
-%module(directors="1") cdapcallbackjava
+
+%module(directors="1") cdapcallbackjava
 
 %{
+#include <iostream>
```

Comment:
We dont want swig to wrap the data-types in iostream, just use the cerr object in the generated native code, to print a warning, if a AP bound IPCEvent has no Java equivalent.